packages/near-fast-auth-signer-e2e-tests/test-app/App.tsx

```diff
@@ -74,7 +74,7 @@ export default function App() {
       )}
       <button
         type="button"
-        data-test-id="sign-transaction-button"
+        data-testid="sign-transaction-button"
         onClick={() => {
           fastAuthWallet
             .signAndSendTransaction(
```

Comment:
should we update this to the new `signAndSendSignedDelegate`? Tests should fail if we send directly to rpc because the created test account has no balance right?

Reply:
This will be done once we publish the new wallet version. 

Currently the `signAndSendTransaction ` still send a `signedDelegate`and the test is passing as you can see on the CI